frontend/styles/components/meeting-form/ResumeMeetingModal.module.scss

```
@@ -0,0 +1,144 @@
+@import '../../Variables.module.scss';
```

Stylelint (17.14.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)